sys/include/net/netopt.h

```diff
@@ -202,6 +202,13 @@ typedef enum {
     NETOPT_CCA_THRESHOLD,
 
     /**
+     * @brief CCA mode for the radio transceiver
+     */
+    NETOPT_CCA_MODE,            /**< get/set the CCA mode as uint8_t
+                                 *   corresponding to the respective PHY standard,
+                                 *   e.g Std 802.15.4 */
```

Comment:
Both of these (on top and below) are ways to write `@brief` tags. Please decide for one of them. And again: please document the type (`netdev2_ieee802154_ccc_mode_t` I presume)